.github/workflows/CI.yml

```diff
@@ -89,8 +89,8 @@ jobs:
           npm --version
       - name: Install yarn
         run: |
-          npm install --verboase --global yarn
```

Comment:
@webwarrior-ws oh wow this is an unfortunate typo; do you mind fixing this as a separate commit please (this way, in case we need to revert this commit later, we would not bring back a typo)

Reply:
Moved to separate commit